silverCrawler/silverCrawler-war/src/main/java/org/silverpeas/components/silvercrawler/control/SilverCrawlerSessionController.java

```diff
@@ -512,8 +493,7 @@ public boolean checkRWSettingsAccess(boolean throwException)
     }
     if (!userAllowedToSetRWAccess) {
       if (throwException) {
-        throw new SilverCrawlerForbiddenActionException(
-            "SilverCrawlerSessionController.checkRWSettingsAccess", SilverpeasException.ERROR,
+        throw new SilverCrawlerForbiddenActionException(CHECK_RWSETTINGS_ACCESS, SilverpeasException.ERROR,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "SilverpeasException"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)